Gemfile

```diff
@@ -7,6 +7,7 @@ gem 'dotenv-rails', require: 'dotenv/rails-now'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 6.0.2', '>= 6.0.2.2'
+gem 'rack', '>= 2.2.3'
```

Comment:
Would `bundle update` not have done this for us without introducing a new explicit dependency?